op-node/rollup/finality/finalizer.go

```diff
@@ -300,6 +292,80 @@ func (fi *Finalizer) tryFinalize() {
 	}
 }
 
+/*
+ *  findLastFinalizedL2BlockWithConsecutiveQuorom tries to find the finalized L2 block.
```

Comment:
tries to find the last finalized L2 block with consecutive quorom

Reply:
fixed in https://github.com/babylonchain/optimism/pull/23/commits/c89dd55766047361ef5e3c04d497be5a17aeee0d